src/Components/SidebarItems/Sidebar.js

```diff
@@ -7,7 +7,7 @@ const Sidebar=()=>{
                 <Link className="sidebarItemsLink"  to="/"><i className="fa fa-home"></i>Home</Link>
                 <Link  className="sidebarItemsLink" to="/VideoListing"><i className="material-icons">explore</i>Explore</Link>
                 <Link className="sidebarItemsLink"  to="/WatchLater"><i className="fa fa-clock-o"></i>Watch later</Link> 
-                <Link  className="sidebarItemsLink" to="/"><i className="fa fa-thumbs-up"></i>Liked Videos</Link>
+                <Link  className="sidebarItemsLink" to="/LikedPage"><i className="fa fa-thumbs-up"></i>Liked Videos</Link>
```

Comment:
better to have the link `urls` as all lowercased

Reply:
okey I will do